Cargo.toml

```diff
@@ -97,6 +97,7 @@ log = "0.4"
 mime = "0.3"
 paste = "1"
 pin-project = "1.0.0"
+pin-project-lite = "0.2.7"
```

Comment:
`pin-project` *and* `pin-project-lite`?

Reply:
we're migrating as we go, both crates are in the dep graph anyway

Reply:
The only blocker of removing `pin-project`besides this macro was the pin drop attr used in logger middleware. 
`pin-project-lite 0.2.7` added this feature so at this point it's possible to remove `pin-project` from actix-web